modules/core/shared/src/main/scala/retry/RetryPolicy.scala

```diff
@@ -97,4 +109,22 @@ object RetryPolicy {
           y: RetryPolicy[M]
       ): RetryPolicy[M] = x.join(y)
     }
+
+  def withShow[M[_]](
```

Comment:
Very minor point but could you move these functions up slightly? I'd like to keep all factory methods at the top of the object and type class instances at the bottom.